components/Launcher.vue

```diff
@@ -8,26 +8,61 @@
         align="center"
       >
         <h4 class="pb-3">Please complete the recaptcha to launch the app</h4>
-        <Recaptcha />
+        <vue-recaptcha
+          ref="recaptcha"
+          :sitekey="site_key"
+          :loadRecaptchaScript="true"
+          @expired="is_captcha_validated = false"
+          @verify="submit_recaptcha"
+          align-self="center"
+        />
       </v-col>
-      <v-col
-        v-else-if="!cloud_store.is_running && cloud_store.is_connexion_launched"
-      >
+      <v-col v-if="!is_cloud_running && is_connexion_launched">
         <Loading />
       </v-col>
     </v-row>
   </v-container>
 </template>
 
 <script setup>
+  import { VueRecaptcha } from "vue-recaptcha"
+
   const websocket_store = use_websocket_store()
   const cloud_store = use_cloud_store()
-  const { is_captcha_validated } = storeToRefs(cloud_store)
+  const { is_cloud_running, is_captcha_validated, is_connexion_launched } =
+    storeToRefs(cloud_store)
+
+  const site_key = useRuntimeConfig().public.RECAPTCHA_SITE_KEY
 
   watch(is_captcha_validated, async (value) => {
     if (value === true && process.client) {
+      console.log("is_captcha_validated 2", is_captcha_validated.value)
```

Comment:
Remove all console.logs

Reply:
I'll do it once all the errors are resolved